packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -31,20 +31,26 @@ export interface TreeEdit {
 	index: number;
 }
 
-export interface FuzzInsert {
-	fuzzType: "insert";
+export type FuzzInsert = FuzzSequenceFieldInsert;
+
+export interface FuzzSequenceFieldInsert {
+	fuzzType: "sequenceFieldInsert";
```

Comment:
Adding a "sequence" prefix to the sequence field operations, implies all edits are going to exist in the same namespace.
It would be better to create a hierarchy to keep things more organized (and modular).
So I think that means we want a flag to describe whether the operation is an edit, synchronize, or a branch operation, then for edits, we want a separate flag (and generator) to describe whether the edit is a node edit, or a field edit, then for field edits we want a separate flag to describe which kind of field it is an edit of, then for each field kind (currently just sequence) we want a flag to describe whether it is an insert or a delete.

Reply:
changed the edit generator hierarchy